linera-rpc/proto/rpc.proto

```diff
@@ -197,19 +197,10 @@ message LiteCertificate {
 
 // A certified statement from the committee, together with other certificates
 // required for execution.
-message Certificate {
+message CertificateRequest {
```

Comment:
Maybe `HandleCertificateRequest`? We're not requesting a certificate, after all.